munch-match/src/components/foodOptionButtons.jsx

```diff
@@ -1,121 +1,132 @@
 //Tells react that this is an component
-import { Component } from "react";
+import {Component} from "react";
 import foodChoices from "../data/categories";
 import FoodOptionButton from "./FoodOptionButton";
 import "../App.css"
 import styles from "./FoodOptionButtons.module.css"
+import quizQuestions from "../data/quizQuestions";
 
 //Creates the Buttons component
-class FoodOptionButtons extends Component{
-  handleClick = (type) => {
-      //generate a random type from the food choice array
-      if (type === "random"){
-          const randomNumber = Math.floor(Math.random() * foodChoices.length);
-          type = foodChoices[randomNumber];
-      }
-        this.props.onButtonClick(type);
+class FoodOptionButtons extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            quizStep: "start", // Initial quiz step
```

Comment:
Is there any way you can get rid of this property by using the `currentQuestionId` instead? You could set it to null and then check whether it is null, less than `steps.length`, or equal to `steps.length`?

Reply:
Will resolve this in another PR just to get the other stuff moving.